chart/templates/manager-rbac.yaml

```diff
@@ -212,6 +212,38 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+    - secrets.hashicorp.com
+  resources:
+    - hcpauths
+  verbs:
+    - create
+    - delete
+    - get
+    - list
+    - patch
+    - update
+    - watch
+- apiGroups:
+    - secrets.hashicorp.com
+  resources:
+    - hcpauths/status
+  verbs:
+    - get
```

Comment:
Does this need `patch` and `update` as in config/rbac/role.yaml?

Reply:
Good catch! I had to merge the hcpauths stuff config/rbac/role.yaml. We will need to look at how we can ensure that the RBAC config is always synced to the chart.

Reply:
Fixed with fb82a562276ec679a7058512783b0156e7b06f9a